.github/workflows/rust-check.yml

```diff
@@ -0,0 +1,36 @@
+name: rust-build
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Install Rust
+        uses: hecrj/setup-rust-action@v2
+        with:
+          rust-version: '1.68.0'
```

Comment:
This is now quite old. Just making sure it is intentional?

Reply:
Yes! The idea is that technically upping the required rust version is a breaking change although we don't have any public MSRV. I chose 1.68 as it is the first one with sparse registry which is faster for CI.